packit/api.py

```diff
@@ -2139,6 +2143,107 @@ def run_osh_build(
 
         return cmd_result.stdout
 
+    def create_obs_project(
+        self,
+        project: str,
+        targets: str,
+        owner: Optional[str],
+        package_config: PackageConfig,
+        description: Optional[str],
+    ):
+        conf.get_config()
+        owner = (
+            owner or conf.config["api_host_options"][self.obs_helper._API_URL]["user"]
+        )
+        prj_name = project or f"home:{owner}:packit"
+
+        targets_list = targets.split(",")
+        for target in targets_list:
+            if target in DEPRECATED_TARGET_MAP:
+                logger.warning(
+                    f"Target '{target}' is deprecated. "
+                    f"Please use '{DEPRECATED_TARGET_MAP[target]}' instead.",
+                )
+
+        prj_meta = self.obs_helper.targets_to_project_meta(
```

Comment:
Here, I would try to be as explicit as possible since this term might not be clear for people outside of OBS.

Maybe, `project_metadata`?